rust/examples/fss_fast.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------
+# CREDITS: This is sourced mostly from:
+# https://github.com/nathan-eize/fss/blob/master/fss_core.py
```

Comment:
There is no license statement that I can see on the original code. Given it's public on github, I would presume the author expects it might get utilised, but it would really be preferable if the original author can clearly license their code. I will not accept code into the repository that has an unclear license status to ensure that the status of the scores repo is not ambiguous in any way.

Reply:
Agree.

I mostly had it in place for verifying my implementation results. So far, I've been getting the same results (but with consistently faster speed compared to fast-fss, unlike numba which has some initialization/startup delays). 

It seems that the author themselves applied parts of the code from the original authors of the papers referenced, so we may be able to ask the original authors. However, it's not necessary to be kept in the repo, we can probably remove it.